### PR TITLE
187 update db table

### PR DESCRIPTION
**Pull request by RauliV**
Edited BUILDS -table,
Store actual build id, instance name, template name,
/api/status GET -> POST + body formation in svelte.App
/api/history POST/GET routes
Catch error when failed and no error data available yet
Test case for getStatus function. (Db tests cause port conflict and those are commented out)

### CODE REVIEW

File: Backend/index.js [hunk 1]
```diff
@@ -48,20 +48,48 @@ async function triggerBuild(buildUrl, options) {
   return response;
 }
 
+//For single row query to get additional information
+app.post('/api/history', (req, res) => {
+  const buildId = req.body.buildId;
+  db.query(`SELECT * FROM BUILDS WHERE build_id = ${buildId}`, function (err, result, fields) {
+    if (err) {
+      console.log(fields);
```
Comment (limpparii) [inline Backend/index.js:56]: I think we should avoid letting console.logs into the production

File: Backend/index.js [hunk 2]
```diff
@@ -48,20 +48,48 @@ async function triggerBuild(buildUrl, options) {
...
+    res.json(result);
+  });
+});
+
+
+//Returns wholde table. Could be narrowed down to needed.
+app.get('/api/history', (req, res) => {
+  db.query('SELECT * FROM BUILDS', function (err, result, fields) {
+  if (err) {
+    console.log(fields);
```
Comment (limpparii) [inline Backend/index.js:69]: Same here

File: Backend/test/index.test.js [hunk 3]
```diff
@@ -7,6 +7,8 @@ import assert from 'assert';
 import app from '../index.js';
 import { indexFactory } from '../index.js';
 import { gitFactory } from '../github.js';
+//import mysql from 'mysql';
```
Comment (limpparii) [inline Backend/test/index.test.js:10]: Is this one still needed?

Comment (limpparii) [reply Backend/test/index.test.js:10]: Nvrmnd, i read rest of the commented out tests :)

**Review by limpparii — APPROVED** [review]
Few console.logs noticed. I think those could be removed or reasoned. Other vise looks ok on code level.

File: Backend/index.js (hunk 2 above)
Comment (RauliV) [reply Backend/index.js:69]: Yeah, these were for eslint to "use" unused variable.  Others are for actual informational purpose - not for testing. I can ofc remove them.

**Review by irrelv — APPROVED** [review]

File: Backend/index.js (hunk 2 above)
Comment (limpparii) [reply Backend/index.js:69]: I think the right solution would be to have a some way to only have the wanted parameters in the function call. If tools, we are using to ensure quality, require as to make a bad solutions, then we have issue with the tool. In many other places the unused parameter check is very usefully so I would rather not remove it from eslint. Was there way to ignore eslint rule in spesific spot in code?

**Pull request merged**